### PR TITLE
auditbeat/module - docs and comment nits

### PR DESCRIPTION
**Pull request by andrewkroh**
## Proposed commit message

Really minor fixes to docs and comments in auditbeat/modules. As I was reading the code I was applying some changes, so I figured I should contribute them.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: auditbeat/module/file_integrity/action.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func (action Action) InOrder(existedBefore, existsNow bool) ActionArray {
 	hasConfigChange := action&ConfigChange != 0
 	hasUpdate := action&Updated != 0
 	hasAttrMod := action&AttributesModified != 0
-	action = Action(int(action) & int(^(ConfigChange | AttributesModified)))
+	action = Action(int(action) & ^(ConfigChange | AttributesModified))
```
Comment (efd6) [inline auditbeat/module/file_integrity/action.go:158]:
```suggestion
	action = Action(int(action) &^ (ConfigChange | AttributesModified))
```
?

File: auditbeat/module/auditd/audit_linux.go [hunk 2]
```diff
@@ -980,7 +980,7 @@ func determineSocketType(c *Config, log *logp.Logger) (string, error) {
 			return "", fmt.Errorf("failed to create audit client: %w", err)
 		}
 		// Ignore errors if a socket type has been specified. It will fail during
-		// further setup and its necessary for unit tests to pass
+		// further setup and is necessary for unit tests to pass.
```
Comment (efd6) [inline auditbeat/module/auditd/audit_linux.go:983]: I'm still not clear what this actually means.

Comment (andrewkroh) [reply auditbeat/module/auditd/audit_linux.go:983]: I agree. Should we maybe remove that second unclear sentence?

Comment (efd6) [reply auditbeat/module/auditd/audit_linux.go:983]: Looking through the code path, I think I understand what is being said here; that we are here to determine the type, but we've already been told it, so if we fail it doesn't matter, and there is a test that depends on this returning nil (this is `TestConfigValidateImmutable` with the socket type "unicast"), so we must let this through. I don't think we are losing much by removing the second sentence.

Comment (andrewkroh) [reply auditbeat/module/auditd/audit_linux.go]: Second sentence removed.  97d5ab5

**Review by efd6 — APPROVED** [review]

**Pull request merged**